design/one-pager-template-stacks.md

```diff
@@ -0,0 +1,344 @@
+# Templates Stacks
+
+* Owner: Marques Johansson (@displague)
+* Reviewers: Crossplane Maintainers
+* Status: Draft, revision 1.0
+
+## Outline
+
+* [Background](#Background)
+* [Terms and Acronyms](#Terms-and-Acronyms)
+* [Benefits of Template Stacks](#Benefits-of-Template-Stacks)
+  * [Simple Variable Substitution](#Simple-Variable-Substitution)
+  * [No Custom Controller Needed](#No-Custom-Controller-Needed)
+  * [Composition](#Composition)
+* [Template Stack Manager](#Template-Stack-Manager)
+  * [Deployment](#Deployment)
+  * [Owner References](#Owner-References)
+  * [RBAC](#RBAC)
+  * [Life Cycle and Updates](#Life-Cycle-and-Updates)
+  * [Unknown Types](#Unknown-Types)
+
+## Background
+
+Early thinking on Stacks included the idea of simplifying the overhead involved in life-cycle management for complex workloads, especially as this relates to consuming Crossplane managed services. However, to contribute a new Stack in the Stack eco-system today, it is necessary to create a new Controller, typically in Go, typically using Kubebuilder.
```

Comment:
nit: "typically in Go, typically using Kubebuilder." -> "typically in Go and using Kubebuilder."